src/main/resources/spring/camel-context.xml

```diff
@@ -155,8 +187,8 @@
             <description>Report REST API</description>
             <get uri="/">
                 <description>Get the list of all the reports for the user</description>
-                <param name="page" type="query" defaultValue="0" required="false" description="Define the page requested" />
-                <param name="size" type="query" defaultValue="10" required="false" description="Define the size of the requested page" />
+                <param name="offset" type="query" defaultValue="0" required="false" description="Define the page requested" />
+                <param name="limit" type="query" defaultValue="10" required="false" description="Define the size of the requested page" />
```

Comment:
The description of these QueryParameters still have the definition of "page". Could you please edit it. Maybe we can reuse this line https://github.com/project-xavier/xavier-integration/blob/e4293da7eac386a3a82f5ad611a851aebce25daf/src/main/resources/spring/camel-context.xml#L242